### DISCUSSION TITLE
Sort directories before files (Linux, like Nemo)

### DISCUSSION THREAD

**Discussion by WillemsenGithub** [doublecmd/doublecmd#1517] [General]
Sort directories, before files, before hidden (.folders)
Linux, like Nemo

**Comment by alexx2000** [comment]
DC sort directories before files by default.

**Reply by WillemsenGithub** [reply]
Thanks for reply. It does, when .hidden folders/files are switched off. (Linux)
When .hidden folders are switched on, it first sort the .hidden folder, then the visible folders, then .hidden files, then files

Is it possible to list alphabetically first the visible folders, then .hidden folder, then files, then .hidden files ?

**Reply by alexx2000** [reply]
No.
